### PR TITLE
feat: update favicons and meta tags

### PR DESCRIPTION
**Pull request by krishnans2006**
## Proposed changes
- Organize favicons into `static/img/favicon`
- Organize meta tags in `meta.html`
- Add and remove favicons to fit modern standards
  - Add png icons with precedence (16x16, 32x32)
  - Add updated favicon.ico (16x16, 32x32, 48x48)
  - Only load a single 180x180 apple-touch-icon
  - Add Safari pinned-tab support (`mask-icon`)
  - Add Windows support (`browserconfig.xml`)
  - Remove unused favicons

## Brief description of rationale
Ion does not follow some best practices for favicons. These changes were inspired by [RealFaviconGenerator](https://realfavicongenerator.net/) and commits like tjcsl/tin@92c325818456f161e608a9eb39828b9de98b406a.

### CODE REVIEW

**Comment by alanzhu0** [comment]
Can you list the visual changes this produces, just for reference and since this is fairly significant?

File: intranet/static/img/favicon/safari-pinned-tab.svg
Comment (alanzhu0) [inline intranet/static/img/favicon/safari-pinned-tab.svg:1]: This one looks all black - not the grey the other icons use. Is that intended?

File: intranet/static/img/favicon/mstile-150x150.png
Comment (alanzhu0) [inline intranet/static/img/favicon/mstile-150x150.png:1]: Also, this one is off center, is that intended?

File: intranet/static/img/favicon/safari-pinned-tab.svg
Comment (krishnans2006) [reply intranet/static/img/favicon/safari-pinned-tab.svg:1]: Yep, it's intended. The Safari pinned tab icon is supposed to be a silhouette and gets colored by the `color` field in `meta.html` line 30.

**Comment by krishnans2006** [comment]
> Can you list the visual changes this produces, just for reference and since this is fairly significant?

Sure. As summarized above, besides speed/quality improvements and re-organization, this PR affects the following visuals:

#### New: Windows 8/10 (Metro Icon)
image

#### New: Safari (Pinned Tab Icon)
image

**Review by alanzhu0 — APPROVED** [review]
Thanks for explanations and LGTM

**Pull request merged**